app/Http/Controllers/Auth/LoginController.php

```diff
@@ -27,13 +32,31 @@ class LoginController extends Controller
      */
     protected $redirectTo = '/';
 
+    private $employeeRepository;
+
     /**
      * Create a new controller instance.
      *
      * @return void
      */
-    public function __construct()
+    public function __construct(EmployeeRepository $employeeRepository)
     {
+        $this->employeeRepository = $employeeRepository;
         $this->middleware('guest', ['except' => 'logout']);
     }
-}
```

Comment:
The middlewares are already there, but i removed the login function all together and redirecting handled the problem.